test/Microsoft.Health.SqlServer.Web/Features/Schema/Model/VLatest.Generated.cs

```diff
@@ -0,0 +1,112 @@
+//------------------------------------------------------------------------------
+// <auto-generated>
+//     This code was generated by a tool.
+//
+//     Changes to this file may cause incorrect behavior and will be lost if
+//     the code is regenerated.
+// </auto-generated>
+//------------------------------------------------------------------------------
+namespace Microsoft.Health.SqlServer.Web.Features.Schema.Model
+{
+    using Microsoft.Health.SqlServer.Features.Client;
+    using Microsoft.Health.SqlServer.Features.Schema.Model;
+
+    internal class VLatest
+    {
+        internal readonly static InsertNumbersProcedure InsertNumbers = new InsertNumbersProcedure();
+        internal readonly static MyProcedureProcedure MyProcedure = new MyProcedureProcedure();
+        internal class InsertNumbersProcedure : StoredProcedure
```

Comment:
Nit: +newline 😄

Reply:
This is formatted using Roslyn's [`NormalizeWhitespace()`](https://docs.microsoft.com/en-us/dotnet/api/microsoft.codeanalysis.syntaxnodeextensions.normalizewhitespace?f1url=%3FappId%3DDev16IDEF1%26l%3DEN-US%26k%3Dk(Microsoft.CodeAnalysis.SyntaxNodeExtensions.NormalizeWhitespace%60%601);k(SolutionItemsProject);k(DevLang-csharp)%26rd%3Dtrue&view=roslyn-dotnet) method, which formats members like this. 
We can write/use another formatter, but I'd avoid including that change with this.

Reply:
Actually, maybe a small change can fix this....

Reply:
Improved the overall formatting [here](https://github.com/microsoft/healthcare-shared-components/pull/130/commits/03534da7309fab973802c3275e7a34de65152edd)